qbraid_qir/qiskit/convert.py

```
@@ -0,0 +1,89 @@
+
+from typing import Optional
+
+from pyqir import Context, Module, qir_module
+from qiskit.circuit import QuantumCircuit
+
+from .elements import QiskitModule, generate_module_id
+from .exceptions import QiskitConversionError
+from .visitor import BasicQiskitVisitor
+
+
+def qiskit_to_qir(circuit: QuantumCircuit, name: Optional[str] = None, **kwargs) -> Module:
```

Ruff (0.15.2)

[warning] Missing type annotation for `**kwargs`

(ANN003)